indico/modules/users/templates/users_admin.html

```diff
@@ -79,11 +87,14 @@ <h3>
                                             {{ user_entry.full_name }}
                                         {%- endif -%}
                                     </span>
+                                    {% if user_entry.user %}
+                                        {{ labels(user_entry.user) }}
+                                    {% endif %}
                                     <span class="right">
```

Comment:
This is no longer right-aligned. It looks especially strange when there's a label for the user as well.